src/NetworkTest/testQuality/index.ts

```diff
@@ -307,16 +312,15 @@ function checkSubscriberQuality(
 
             subscriberMOS(builder.state, subscriber, publisher, getStatsListener, resultsCallback);
 
-            // We add +1 to the testTimeout value in order to consider the last stats snapshot.
-            mosEstimatorTimeoutId = window.setTimeout(processResults, testTimeout + 1);
+            mosEstimatorTimeoutId = window.setTimeout(processResults, testTimeout);
 
             window.clearTimeout(stopTestTimeoutId);
             stopTestTimeoutId = window.setTimeout(() => {
               stopTestTimeoutCompleted = true;
               if (stopTestCalled && stopTest) {
                 stopTest();
               }
-            }, 5000);
+            }, testTimeout + 1);
```

Comment:
I remember you asking about this. The readme says "The test will not stop until it has been
running for at least 5 seconds (after the user has granted access to the camera and microphone)." Is this a change that needs to be documented in the readme?